service-rpc/if/TCLIService.thrift

```diff
@@ -751,6 +751,52 @@ struct TGetTypeInfoResp {
   2: optional TOperationHandle operationHandle
 }
 
+// UploadData()
+//
+// UploadData data to table/path.
+struct TUploadDataReq {
+  // The session to execute the statement against
+  1: required TSessionHandle sessionHandle
+
+  // The table to be stored
+  2: optional string tableName
+
+  // The path to be stored
+  3: optional string path
+
+  // The data to be transferred
+  4: required binary values
+}
+
+struct TUploadDataResp {
+  1: required TStatus status
+  2: required TOperationHandle operationHandle
+}
+
+// DownloadData()
+//
+// Download data to JDBC client.
+struct TDownloadDataReq {
+  // The session to download data
+  1: required TSessionHandle sessionHandle
+
+  // The download table name
+  2: optional TPatternOrIdentifier tableName
+
+  // The download query
+  3: optional string query
+
+  // The download file format
+  4: optional string format
```

Comment:
Curious how is the file format being used in downloading data?

Reply:
For example, we want to download data as csv format, we don't need to care about the table format:
```scala
spark.table("tableName").write.format("csv").option("dateFormat", "yyyy-MM-dd").save("/tmp/spark/download/sessionid/queryid")
```
and then download data from `"/tmp/spark/download/sessionid/queryid"`.